### DISCUSSION TITLE
Organic Maps | OSMAnd

### DISCUSSION THREAD

**Discussion by Simplenuity** [organicmaps/organicmaps#6029] [Q&A] [answered]
Apologizes if this has been outlined somewhere but I couldn't find during my search.

I'd like to understand **how Organic Maps distinguishes itself from OSMAnd?**

What are the distinctive features/philosophies that prevent both projects bundle their resources to provide an open source navigation software utilizing OpenStreetMaps?

**Comment by biodranik** [comment, marked as answer]
Why are you asking? Is your goal to save some disk space by sharing maps between apps?

1. Organic Maps is fast, simple to use, and privacy-focused in the first place. We'll keep these principles at the top of our priorities.
2. Orgnic Maps is fast because we have our specialized maps data format. We don't mind if OSMand decides to use it, but we won't use slow OSMand's format for obvious reasons.
3. Organic Maps is run by donations and is built by the community.

**Comment by Simplenuity** [comment]
Thank you for your detailed response!

There can obviously be many reasons why people/groups decide to develop similar apps in parallel. Be it technological, economical, political, philosophical, or personal to mention a few. If the reason isn't obvious to me from the information I find, I rather prefer to ask before making any assumptions :)

I was not aware of that the maps are not generally used as provided by the OSM project and that you use a specialized data format instead!

Even if I didn't ask with the intention to share map data between the apps, it's clear now, it wouldn't even be possible.

As you mention privacy focus as one of the distinctive features I'll have a closer look into the practice of OSMand.

Thanks again and keep up the great work!

**Comment by hb0nd** [comment]
> As you mention privacy focus as one of the distinctive features I'll have a closer look into the practice of OSMand.

OSMAnd is also privacy focussed :)

the main  difference between OSMAnd & OM is that OSMAnd takes an "everything and the kitchen sink" approach, cramming as much functionality into the app as possible, often to the detriment of usability & performance.

Organic Maps intends to "keep it stupidly simple" and provide a clean and user-experience focussed app, which is as optimised and performant as possible

both projects have their place, and can learn from each other :)

**Comment by Simplenuity** [comment]
FYI
Just came across the following, interesting article. Original unfortunately in German.

Original:

> [Wie navigierst Du? Karten-Apps: Von "Datenschleudern" und "Funktionsmonstern"](https://gnulinux.ch/wie-navigierst-du-karten-apps-von-datenschleudern-und-funktionsmonstern)
In einer kleinen Umfrage haben wir gefragt, welche Karten-App Ihr nutzt und vor allem: Warum? Was zeichnet die verschiedenen Apps aus und welche Apps können für verschiedene individuelle Anforderungen empfehlenswert sein?

Google Translate:

> [How do you navigate? Map apps: About “data slingers” and “functional monsters”](https://gnulinux-ch.translate.goog/wie-navigierst-du-karten-apps-von-datenschleudern-und-funktionsmonstern?_x_tr_sl=auto&_x_tr_tl=en&_x_tr_hl=en-US&_x_tr_pto=wapp)
In a small survey we asked which map app you use and, above all, why? What distinguishes the different apps and which apps can be recommended for different individual requirements?
